parcels/collection/collections.py

```diff
@@ -688,6 +784,7 @@ def merge(self, same_class=None):
         This function merge two strictly equally-structured ParticleCollections into one. This can be, for example,
         quite handy to merge two particle subsets that - due to continuous removal - become too small to be effective.
 
+        TODO - RETHINK IF TAHT IS STILL THE WAY TO GO:
```

Comment:
This either needs to be implemented before the PR merge, or expanded/explained better for follow-up